eng/pipelines/templates/variables/globals.yml

```diff
@@ -2,4 +2,8 @@ variables:
   DotNetCoreSDKVersion: '3.0.100-preview5-011568'
   DotNetCoreRuntimeVersion: '2.1.10'
   OfficialBuildId: $(Build.BuildNumber)
-  skipComponentGovernanceDetection: true
\ No newline at end of file
+  skipComponentGovernanceDetection: true
+  ${{ if ne(variables['System.TeamProject'], 'internal') }}:
+    BuildConfiguration: ''
```

Comment:
I agree this approach is likely better then the other one. However did you seem my comment about setting this to Debug and not empty? I think we should always set it to either Debug or Release.

Reply:
Ok, I can set it to debug. However it works the way it is right now in both playground and internal projects